envoimoinscher.php

```diff
@@ -2473,6 +2708,7 @@ public function hookupdateCarrier($params)
 	 */
 	public function hookHeader()
 	{
+		$this->getContext()->controller->addJs('https://maps.google.com/maps/api/js?sensor=false');
```

Comment:
Is it possible to have it in the hook hookDisplayCarrierList ? It will be better to load it only when needed.